ios-base.xcodeproj/xcshareddata/xcschemes/ios-base-develop.xcscheme

```diff
@@ -108,6 +108,12 @@
             ReferencedContainer = "container:ios-base.xcodeproj">
          </BuildableReference>
       </BuildableProductRunnable>
+      <CommandLineArguments>
+         <CommandLineArgument
+            argument = "-FIRAnalyticsDebugEnabled "
```

Comment:
It is ok to have the flag ending in a space?

Reply:
Yes, but i'll remove it to keep it cleaner